src/utils/createGenerateClassName.js

```diff
@@ -32,11 +32,12 @@ export default (): generateClassName => {
       )
     }
 
+    const prefix = sheet ? (sheet.options.classNamePrefix || '') : ''
```

Comment:
We could put 'c' as a default prefix if user hasn't provided one

Reply:
We can. I thought, `c` is analogue to `rule.key` in development, but not prefix.

Reply:
We added "c" because class names can't start with some characters like number, to ensure that. Should have added a comment. In case where user has defined a prefix, I think we can rely on it.